code/modules/projectiles/guns/projectile/constructable/blunderbuss.dm

```diff
@@ -97,7 +97,7 @@
 	for(var/i=1, i<=prohibited_items.len, i++)
```

Comment:
I wrote it because I didn't know about is_type_in_list.
I'll fix it tomorrow if you really want me to.